src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3273,6 +3329,49 @@ components:
           description: Date from which to remove older log files from repository.
           nullable: true
           example: "2022-07-16"
+    ScheduleBulkScan:
+      type: object
+      properties:
+        bulkActions:
+          type: array
+          items:
+            type: object
+            properties:
+              licenseId:
+                type: integer
+                example: 1
```

Comment:
Can we use shortname here and use LicenseDAO to get the license ID in controller?